student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,8 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+
+    const char* user_name = my_username(); 
+    char* mallocd_user_name = malloc_username_from_conf_file();
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(user_name, mallocd_user_name, "Username is invalid. Check for errors.");
 }
```

Comment:
@sankalp123321 what happens if you don't free the memory you malloced for user name?

Reply:
That memory will remain unoccupied throughout the lifetime of the process. As soon as the program exists, the memory will be restored. 

Shall I reupload the code with free()?